modules/apps/item-selector/item-selector-web/src/main/java/com/liferay/item/selector/web/internal/portlet/LocalizedItemSelectorRendering.java

```diff
@@ -115,6 +115,15 @@ public ItemSelectorViewRenderer getSelectedItemSelectorViewRenderer() {
 		return _itemSelectorViewRenderers.get(_selectedNavigationItemLabel);
 	}
 
+	public NavigationItem getActiveNavigationItem(){
```

Comment:
@Stephy6, podemos calcularlo en
 
`public void add(ItemSelectorViewRenderer itemSelectorViewRenderer) {`

como te mencionaba, como lo hacemos con _selectedNavigationItemLabel

Reply:
Understood, thank you!